docs/infra/api-markdown-documenter/api-documentation-layout.mjs

```diff
@@ -298,13 +298,13 @@ function createDeprecationNoticeSection(apiItem, config) {
 		return undefined;
 	}
 
-	const transformedDeprecatedBlock = transformTsdocNode(deprecatedBlock, apiItem, config);
-	if (transformedDeprecatedBlock === undefined) {
+	const transformedDeprecatedBlockContents = transformTsdoc(deprecatedBlock, apiItem, config);
```

Comment:
Note: the output of `transformTsdoc` is now an array of `BlockContent`, rather than a single `Section` node.